### PR TITLE
Enable dependabot for this repo

### PR DESCRIPTION
**Pull request by mmcgehee-ccc**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "monthly"
```
Comment (zredinger-ccc) [inline .github/dependabot.yml:11]: You may have been out but we talked about doing a 6-month interval for dependency upgrades as it gets to be a bit much monthly.

Comment (mmcgehee-ccc) [reply .github/dependabot.yml:11]: I was there for that but I looked into it and unfortunately it appears that the only intervals supported by dependabot are:
daily, weekly, and monthly (unless I'm missing something).

Comment (mmcgehee-ccc) [reply .github/dependabot.yml:11]: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduleinterval

Comment (zredinger-ccc) [reply .github/dependabot.yml:11]: Bummer

**Review by zredinger-ccc — APPROVED** [review]

**Pull request merged**